website/events/admin/forms.py

```diff
@@ -49,3 +50,31 @@ class Meta:
         widgets = {
             "type": forms.Select,
         }
+
+
+class EventAdminForm(forms.ModelForm):
+    def clean(self):
+        super().clean()
+        self.instance.clean_changes(self.changed_data)
+
+    def clean_organisers(self):
+        if self.request.user.is_superuser or self.request.user.has_perm(
```

Comment:
There should be a 'is_organiser' helper function that you can use so that the task of deciding that is in that helper instead of here.

Reply:
A different action is needed when the user has event_override, hence that function isn't used